app/views/customers/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= @customer.full_name %>
+<%= @customer.cpf %>
+<%= l @customer.birth_date %>
+<%= @customer.rg %>
+<%= @customer.rg_sender %>
+<%= l @customer.rg_expedition %>
+<%= @customer.place_birth %>
+<%= @customer.place_birth_uf %>
+<%= @customer.nacionality %>
+<%= @customer.mother %>
+<%= @customer.father %>
+<%= @customer.postal_code %>
+<%= @customer.address %>
+<%= @customer.neighborhood %>
+<%= @customer.adjunct %>
+<%= @customer.city %>
+<%= @customer.state %>
+<%= @customer.phone %>
+<%= @customer.cell_phone %>
+<%= @customer.email %>
```

Comment:
There is a gem called show_for, I enjoy it to use in that some case of simple_form.